docs/DeveloperGuide.md

```diff
@@ -175,26 +175,31 @@ How the parsing works:
 
 **API** : [`Model.java`](https://github.com/AY2122S1-CS2103T-T15-1/tp/tree/master/src/main/java/gomedic/model/Model.java)
 
-<img src="images/ModelClassDiagram.png" width="450" />
-
+![ModelClassDiagram](images/ModelClassDiagram.png)
+![EntryModelClassDiagram](images/EntryModelClassDiagram.png)
 
```

Comment:
By the way, just some bugs in your diagram, 
1. The id should be {abstract} 
2. You forgot the multiplicity for each id

Reply:
actl right should i just delete all the ids and just put one under person, coz idk how to show the relationship between person and id and make it not seem like double counting since patient and doctor both inherit from person

Reply:
Yeah that should be fine actually

Reply:
okie